### PR TITLE
Copy documentation

### PR DESCRIPTION
**Pull request by kbaley**
Copied the doco and samples from https://docs.particular.net/persistence/mongodb-tekmaven/ in anticipation for the upcoming beta release of the official package.

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * [Timeouts](https://docs.particular.net/nservicebus/sagas/timeouts)
+ * Subscriptions
+ * [Sagas](https://docs.particular.net/nservicebus/sagas/)
+
+
+## Usage
+
+To use MongoDB for persistence
+
+snippet: MongoUsage
```
Comment (timbussmann) [inline docs/readme.md]: we need to inline the snippets

File: samples/databus/sample.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
```
Comment (timbussmann) [inline samples/databus/sample.md]: the headers can be removed

File: samples/databus/sample.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ * Messages - A class library containing the sample messages. Only one of the message types utilizes the DataBus.
+ * Sender - A console application responsible for sending the large messages.
+ * Receiver - A console application responsible for receiving the large messages from Sender.
+
+
+### Messages project
+
+There are two messages defined in the Messages project. Start by looking at `AnotherMessageWithLargePayload` which is not utilizing the DataBus mechanism. The message is a simple byte array command:
+
+snippet: AnotherMessageWithLargePayload
```
Comment (timbussmann) [inline samples/databus/sample.md]: snippets need to be inlined

File: samples/databus/sample.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+snippet: MessageWithLargePayload
+
+
+### Configuring the Databus location
+
+Both the `Sender` and `Receive` project need to share a common location to store large binary objects.
+
+snippet: ConfigureDataBus
+
+Note that the connection string used for the databus is shared by the [MongoDB Persistence](/persistence/mongodb-tekmaven).
```
Comment (timbussmann) [inline samples/databus/sample.md]: link url needs to be the full url

File: samples/simple/MongoTekmaven_8/Client/Client.csproj [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (timbussmann) [inline samples/simple/MongoTekmaven_8/Client/Client.csproj]: Let's just keep the v9 version of that sample?

Comment (timbussmann) [reply samples/simple/MongoTekmaven_8/Client/Client.csproj]: We could also remove the version related folder to simplify the path a bit as this won't be necessary when the sample is versioned along with the sources

File: samples/simple/sample.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
+---
```
Comment (timbussmann) [inline samples/simple/sample.md]: headers not needed

File: samples/simple/sample.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+ * When the `CompleteOrder` timeout fires the `OrderSaga` publishes an `OrderCompleted` event.
+ * `Server` then publishes a message that the client subscribes to.
+ * `Client` handles the `OrderCompleted` event.
+
+
+### MongoDB configuration
+
+The `Server` endpoint is configured to use the MongoDB persistence with a connection string of `mongodb://localhost:27017/SamplesMongoDBServer`.
+
+snippet: MongoDBConfig
```
Comment (timbussmann) [inline samples/simple/sample.md]: inline snippets

File: samples/simple/MongoTekmaven_8/Client/Client.csproj (hunk 5 above)
Comment (kbaley) [reply samples/simple/MongoTekmaven_8/Client/Client.csproj]: I'll remove the v8 version but would like to keep the subfolder so the code doesn't get mixed with the .md file and the related .pngs

**Comment by kbaley** [comment]
@timbussmann Updated with your suggestions. I also found another [doco page](https://docs.particular.net/nservicebus/messaging/databus/mongodb-tekmaven) that I moved over.

**Pull request merged**